lib/classes/CLI.js

```diff
@@ -83,6 +84,14 @@ class CLI {
     }
   }
 
+  isConfigIndependentRequest(processedInput) {
```

Comment:
I know I proposed `isConfigIndependentRequest`, but probably `isServiceIndependentCommand` would be more accurate name (sorry for confusion)